server/src/Api/Controllers/GameController.cs

```diff
@@ -22,4 +27,22 @@ public Task<int> CalculateScore(CalculateScoreCommand command)
     {
         return _mediator.Send(command);
     }
+    
+    [HttpPost(Routes.Game.Create)]
+    public Task<Result> CreateGame(string gameName)
```

Comment:
This is `Game`Controller and no need to specify and dublicate `Game` in name of methods.
CreateGame -> Create
GetAllGames -> GetAll